scripts.js

```diff
@@ -229,6 +229,15 @@ function details(institution) {
             studentName = document.createElement('p'),
             studentMajor = document.createElement('p');
         studentPhoto.src = 'portraits/' + currentYear + '/' + student.name + '.jpg';
+        imageExists(studentPhoto.src, function(exists) {
+            if(exists == true)
```

Comment:
‼️

Reply:
You can probably even make this whole thing a ternary statement. At the very least, replace with `if (exists) {`